lib/vdsm/tool/configurators/libvirt.py

```diff
@@ -42,6 +42,7 @@
 from vdsm import constants
 
 
+_LIBVIRT_CONF_PATH = "/etc/sysconfig/libvirtd"
```

Comment:
We already have that path defined under the `LDCONF` key (line ~360) in a better way - it uses the `constants` module. Let's move that definition here, and reuse the new variable in `LDCONF`